### PR TITLE
Changed time formatter

### PR DESCRIPTION
**Pull request by scribe**

### CODE REVIEW

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/DateTimeUtils.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 public final class DateTimeUtils {
 
-    private static final DateTimeFormatter DEFAULT_FORMAT = DateTimeFormatter.ISO_LOCAL_DATE_TIME;
+    private static final DateTimeFormatter DEFAULT_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/DateTimeUtils.java]: Should we include the time zone?  Or is this what was being displayed before?

Comment (scribe) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/DateTimeUtils.java]: Before it was hard coded, Java is super flexible so I we can make it whatever we want.

Comment (rpmoore) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/DateTimeUtils.java]: I guess my question is, does the new format match the old hard coded value?

**Comment by rpmoore** [comment]
This now has a conflict.

**Pull request merged**